### ISSUE TITLE
Good position of Effect panel with RPG Styled Ui 

### ISSUE THREAD

**Issue by Sasmira** [DFreds/dfreds-effects-panel#18]
Hello Hello 

Actually your panel effect is in 310px and it's the good position for your panel if you play with PF2 system, because in PF2, the sidebar is a bit smaller than the standard sidebar
 
```java
.effect-panel {
    position: fixed;
    right: 310px;
    top: 5px;
}
```
But for a standard sidebar, the good value is 350px ;)

```java
.effect-panel {
    position: fixed;
    right: 350px;
    top: 5px;
}
```

S.

**Comment by Sasmira** [comment]
I forgot to precise with RPG Styled Ui ....

**Comment by DFreds** [comment]
Okay. Thanks for the heads up. I have plans to add functionality for dragging the panel around which should fix this

**Comment by Sasmira** [comment]
> Okay. Thanks for the heads up. I have plans to add functionality for dragging the panel around which should fix this

Hoo nice feature !

**Comment by JhaelenTerear** [comment]
Same problem here, it's better now that you can drag&drop verticaly, but it will be better with horizontal drag&drop to !
![image](https://user-images.githubusercontent.com/58259514/162895462-80eb7803-dee5-479f-97b2-6434af52fa34.png)

**Comment by DFreds** [comment]
Fixed in v1.5.1
